terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -62,7 +62,7 @@ locals {
 
   bcl_convert_wfl_version = {
     dev  = "3.7.5"
-    prod = "3.7.5--fade666"
+    prod = "3.7.5--f1e67a3"
```

Comment:
This looks like it has regressed from fade666 (which was on 10 Feb 2022) to f1e67a3 (which was on 7 June 2021)

Reply:
This is what was set in your PR in the `main.tf` file. I basically took your changes and added WTS workflow input changes.

Reply:
Ah okay, so it would have been the old version because the dragen workflow prod branch didn't update the bcl convert, that was done by https://github.com/umccr/infrastructure/pull/198, and the dragen workflow prod branch stemmed from master before that was merged in.

Reply:
But by setting it here it's seen as a change in the file commit which is upstream of #198.